src/HasProfilePhoto.php

```diff
@@ -56,19 +57,55 @@ public function getProfilePhotoUrlAttribute()
     {
         return $this->profile_photo_path
                     ? Storage::disk($this->profilePhotoDisk())->url($this->profile_photo_path)
-                    : $this->defaultProfilePhotoUrl();
+                    : '';
```

Comment:
It's a breaking change.

Reply:
Thats why I have updated both livewire and inertia.
But I guess you are right, users already having jetstream installed will not get the default template updates.

I will think about something.